### DISCUSSION TITLE
Annuler un combat automatiquement si l'un des deux joueurs n'a pas réagit durant le combat

### DISCUSSION THREAD

**Discussion by Bombafox** [Crownicles/Crownicles#1851] [Nouvelles idées]
Ce serait sympa que le bot compte les 30s sans réponse à partir du moment où les 2 personnes on réac au fight 1 fois chacun.

Ça éviterait ce genre de problème ou tu perd alors que c'était même pas voulu.
Genre bug de co, ou juste t'es parti de devant le salon parce que personnes ne réac et que tu va voir sur un autre serv si y a pas des gens qui veulent fight.

Surtout depuis qu'on peut perdre des point de gloire.
Parce que a la limite en amical en s'en fou un peu vu que y a pas de risque.

**Comment by romain22222** [comment]
bon @BastLast je pense que celle ci est à accepter directement, en plus si c'est bien codé derrière, y'a pas grand chose à faire. J'en vois passer de temps en temps des vols involontaires, et avec la gloire ca devient critique

**Comment by BastLast** [comment]
sauf que ça va faire potentiellement des combats très longs...

**Comment by BastLast** [comment]
en fait je suis pas sur d'avoir bien compris la suggestion?

**Reply by BananePlantain** [reply]
de pas perdre de l'énergie bétement parce que l'un des combattants n'as pas pu répondre la 1er fois
